includes/Classifai/Providers/Watson/NLU.php

```diff
@@ -122,11 +123,7 @@ public function reset_settings() {
 	 * @return bool
 	 */
 	public function can_register() {
```

Comment:
An optional nitpick: Maybe we can remove this method? As it is similar to the parent class `Provider`.